extract_wheels/lib/namespace_pkgs.py

```diff
@@ -1,36 +1,11 @@
 """Utility functions to discover python package types"""
 import os
-import sys
 import textwrap
 from typing import Set, List, Optional
 
 from extract_wheels.lib import wheel
 
 
-def pkg_resources_style_namespace_packages(wheel_dir: str) -> Set[str]:
-    """Discovers namespace packages implemented using the 'pkg_resources-style namespace packages' method.
-
-    "While this approach is no longer recommended, it is widely present in most existing namespace packages." - PyPA
-    See https://packaging.python.org/guides/packaging-namespace-packages/#pkg-resources-style-namespace-packages
-    """
-    namespace_pkg_dirs = set()
-
-    dist_info = wheel.get_dist_info(wheel_dir)
-    namespace_packages_record_file = os.path.join(dist_info, "namespace_packages.txt")
-    if os.path.exists(namespace_packages_record_file):
-        with open(namespace_packages_record_file) as nspkg:
-            for line in nspkg.readlines():
-                namespace = line.strip().replace(".", os.sep)
-                if namespace:
-                    namespace_pkg_dirs.add(os.path.join(wheel_dir, namespace))
-    return namespace_pkg_dirs
```

Comment:
This is a no-op:

If a directory specified in `namespace_packages.txt`:
* Is empty, then we aren't going to create it
* Has no `__init__.py`, will be automatically handled by implicit logic
* Has non-empty `__init__.py`, we assume is pkg-util or pkg-resource style, we already skip non-empty init files.